### PR TITLE
Fix cp path in Musca / TF-M build

### PR DESCRIPTION
**Pull request by thomas-fossati**

### CODE REVIEW

**Review by dbkinder — CHANGES_REQUESTED** [review]
The change to the doc is OK, but there are issues with the commit message that needs fixing: the PR title needs to be formatted correctly, and needs a Signed-off-by line (inserted by using "git commit -s", assuming you're got your email and user name set up in git.

File: boards/arm/v2m_musca/doc/index.rst [hunk 1]
```diff
@@ -374,7 +374,7 @@ for the secure veneers.
 
 .. code-block:: bash
 
-   cp -r install/ $ZEPHYR_PROJECT/src/ext
```
Comment (karl-zh) [inline boards/arm/v2m_musca/doc/index.rst]: Hi

Just curious was it led some function errors when having the slash, or just make the alignment?

This change is fine to me, thanks for figure this out.

Please be aware that we are going to include TFM repo and compile TFM automatically when building Zephyr. https://github.com/zephyrproject-rtos/zephyr/pull/19985

Comment (thomas-fossati) [reply boards/arm/v2m_musca/doc/index.rst]: Hi @karl-zh , the problem is that with the extra '/' the path to the veneers object on the subsequent line is off by one path segment.  Not a big issue, one can easily figure that out :-)

In all cases, thanks for the pointer to #19985.  I see some excellent stuff there, which is going to save a lot of time and silly mistakes.  Thanks!

Comment (karl-zh) [reply boards/arm/v2m_musca/doc/index.rst]: I see, thank you to have this fixed. Please give more description in your commit message, and follow what @dbkinder mentioned.

**Review by microbuilder — APPROVED** [review]
The change itself is valid removing the trailing `/` in the documentation.

**Comment by galak** [comment]
@thomas-fossati any plans to fix the git commit issues flag here?

**Pull request closed without merging**